src/main/kotlin/anttracker/MainMenuScreen.kt

```diff
@@ -1,24 +1,18 @@
 package anttracker
 
-val mainMenu =
-    screenWithMenu {
-        title("Main menu")
-        option("Issues") { issuesMenu }
-        option("Requests") { requestsMenu }
-        option("Products") { productsMenu }
-        option("Contacts") { contactsMenu }
-        content { t -> t.printLine("This is the main menu") }
+val requestsMenu =
+    object : Screen {
+        override fun run(): Screen? {
+            TODO("Not yet implemented")
```

Comment:
This is implemented, in Request.kt, in my PR - it will be a function in and of itself

Reply:
This was just to illustrate how to use it. Once we agree, I will change it.